### ISSUE TITLE
Bounty / Contract Based Poll Creation, Fund Distribution, and Development.

### ISSUE THREAD

**Issue by jring-o** [gridcoin-community/Gridcoin-Tasks#187]
Will be adding to this as I flesh it out.

Contract based
Poll created once bounty level (or other contract requirement) reached
Bounty used to fund development
Distribution based on contract

**Comment by Mercosity** [comment]
Excellent ..

**Comment by jring-o** [comment]
The structure I intend to propose is nearing the point where I am comfortable presenting it for feedback.  As I solidify it, I would love some feedback on the initial premise.  Thanks in advance!  

We need a development structure which allows individuals or entities to gauge and receive support (financial or otherwise) for a product from the initial idea to the proposal to the development to the implementation.  The structure must:

1. Allow and encourage collaboration in the open-source style.
2. Allow and encourage the creation of reward substructures (contracts) that are transparent and auditable.
3. Discourage abuse of power, manipulation, and the creation of proposed developments purely for the profit of the developer, an individual, a group of people, or an entity. 
4. Encourage auditable products.
5. Allow and encourage competing ideas to be developed and funded in parallel.
6. Can be implemented in conjunction with a decentralized funding structure, like automated stake/fee splitting (what I've called side-staking in the past.  I will be writing more about this funding structure in a separate issue after posting more details on this development idea).

What else do you think a development structure should seek to do?
Do you think any of these points are wrong or misguided?
What other problems do you see with our current development structures?
Do you know of any development structures which already exist that might be relevant to this discussion (from recent crypto communities to ancient history)?

**Comment by jring-o** [comment]
Moved to #202 

Closing Issue.
